zstacklib/zstacklib/utils/sanlock.py

```
@@ -441,47 +575,140 @@ def get_lockspace(vg_uuid):
+    linux.write_file(bk_file, jsonobject.dumps(context), create_if_not_exist=True)
+
+
+def read_lockspace_metadata_from_backup(vg_uuid, host_id=None):
+    # type: (str, str) -> list[DeltaLease]
+    bk_file = os.path.join(SANLOCK_BACKUP_PATH, "%s.lockspace" % vg_uuid)
+    meta_backup = linux.read_file(bk_file)
+    if not meta_backup:
+        return []
+    try:
+        meta_backup = jsonobject.loads(meta_backup)
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)